src/core.c/IO/Notification.pm6

```diff
@@ -13,6 +13,15 @@ my class IO::Notification {
             "$.path: $.event";
         }
         method IO { $!path.IO }
+
+        multi method WHICH(Change:D: --> ValueObjAt:D) {
+            nqp::box_s(
+              nqp::join('|',nqp::list_s(
+                self.^name,nqp::substr($!event.Str,4),$!path
```

Comment:
Not sure the `substr` is a net win here?

Reply:
I felt the "File" part of the enum was redundant.  Not a biggie, really.